### PR TITLE
[GSoC2024] add test case for subsequent annotation import

### PR DESCRIPTION
**Pull request by umangapatel123**
### Motivation and context

Test Case for the PR #7599 and Issue #7428

### Checklist

- [x] I submit my changes into the `develop` branch
- ~~[ ] I have created a changelog fragment~~ 
- ~~[ ] I have updated the documentation accordingly~~
- [x] I have added tests to cover my changes
- [x] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- ~~[ ] I have increased versions of npm packages if it is necessary~~
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: tests/cypress/e2e/issues_prs2/issue_7428_importing_annotation_from_cloud_after_local_import.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            cy.get('.cvat-modal-export-task').find('.cvat-modal-export-select').should('contain.text', exportFormat);
+            cy.get('.cvat-modal-export-task').find('.cvat-modal-export-filename-input').type(annotationsArchiveNameLocal);
+            cy.contains('button', 'OK').click();
+            cy.get('.cvat-notification-notice-export-task-start').should('be.visible');
+            cy.closeNotification('.cvat-notification-notice-export-task-start');
+            cy.waitForDownload();
+        });
+
+        it('Export Annotation to the cloud storage', () => {
+            cy.clickInTaskMenu('Export task dataset', true, taskName);
```
Comment (novda) [inline tests/cypress/e2e/issues_prs2/issue_7428_importing_annotation_from_cloud_after_local_import.js:64]: There a little bit repeating code in export task datasets. You can use the `exportTask` from [here](https://github.com/opencv/cvat/blob/develop/tests/cypress/support/commands.js#L1169) for exporting task datasets.

It seems the `exportTask` command need modify in export to the cloud storage, if you have the opportunity, let's fix it and be DRY :)

File: tests/cypress/e2e/issues_prs2/issue_7428_importing_annotation_from_cloud_after_local_import.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                cy.get('.cvat-search-target-storage-cloud-storage-field').click();
+                cy.get('.cvat-cloud-storage-select-provider').click();
+            }
+            cy.contains('button', 'OK').click();
+            cy.get('.cvat-notification-notice-export-task-start').should('be.visible');
+            cy.closeNotification('.cvat-notification-notice-export-task-start');
+            cy.waitForFileUploadToCloudStorage();
+        });
+
+        it('Import Annotation from the local storage', () => {
```
Comment (novda) [inline tests/cypress/e2e/issues_prs2/issue_7428_importing_annotation_from_cloud_after_local_import.js:85]: Also in the import datasets, you can use `uploadAnnotations` from [commands](https://github.com/opencv/cvat/blob/develop/tests/cypress/support/commands.js#L858)

Comment (umangapatel123) [reply tests/cypress/e2e/issues_prs2/issue_7428_importing_annotation_from_cloud_after_local_import.js:118]: I tried using the `uploadAnnotations` but the problem is that `uploadAnnotations` is made assuming the annotation is uploaded from the jobs page. so the GET request is intercepted and checked it's status but if you upload annotation from the tasks page, there is no GET request. It only does POST and PUT requests. Hence I have not used `uploadAnnotations` but tried to write modularized code.

**Comment by umangapatel123** [comment]
@novda can you review it?

**Comment by umangapatel123** [comment]
@novda any suggestion, please? I don't know why the check is failing for `rest_api_testing`.

**Comment by novda** [comment]
@umangapatel123 There were some minor problems in ci/cd, now it's okay. Thanks for the contributing!

**Review by novda — APPROVED** [review]

**Pull request merged**